index.html

```diff
@@ -44,19 +52,24 @@ <h3 id="subhero">I am a Software Developer</h3>
   <!-- Work Section -->
 
   <section id="work">  
-    <h2 id="center">My Recent Works</h2>
+    <div id="work-subt">
+      <h2 id="center">My Recent Works</h2>
+      <img src="/img/icons/Vector.png" alt="" class="vector">
+    </div>
     <div class="grid-container">
       <div id="grid-item1">
-        <img src="/img/Img-placeholder.png" alt="">
-        <h2 id="subtitle">Multi-Post Stories</h2>
-        <p id="text-content">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
-        <ul id="lang-boxes">
-          <li class="lang-box1">css</li>
-          <li class="lang-box1">html</li>
-          <li class="lang-box1">bootstrap</li>
-          <li class="lang-box1">Ruby</li>
-        </ul>
-        <button class="main-button">See Project</button>
+        <img class="grid-span" src="/img/Img-placeholder.png" alt="">
+        <div class="grid-content">
+          <h2 id="subtitle">Multi-Post Stories</h2>
+          <p id="text-content">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+          <ul id="lang-boxes">
+            <li class="lang-box1">css</li>
+            <li class="lang-box1">html</li>
+            <li class="lang-box1">bootstrap</li>
+            <li class="lang-box1">Ruby</li>
+          </ul>
+          <button class="main-button">See Project</button>
+        </div>
```

Comment:
Kindly try to fix your image src link right now I can't see them you can fix them by adding a dot before the slash and use the right path
![image](https://user-images.githubusercontent.com/61790587/165863179-f4aeb5c7-c312-4961-a6e8-c7236b2b9450.png)